### PR TITLE
Added a pull request template

### PR DESCRIPTION
**Pull request by kanak22**
I have added a Pull request template for this project. 
This PR closes issue number : #29

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+What sort of change have you made:
+<!--
+Example how to mark a checkbox:-
+- [x] My code follows the code style of this project.
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Code style update (formatting, local variables)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
[the remark this answers is not in the document]
Comment (kanak22) [reply .github/pull_request_template.md:22]: sure

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Checklist:
+<!--
+Example how to mark a checkbox:-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly whereever it was hard to understand.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes have been merged and published in downstream modules.
```
[the remark this answers is not in the document]
Comment (kanak22) [reply .github/pull_request_template.md:36]: I have verified/tested my code by running it locally.
 so I have to add this one line and remove those 4 . Right?

[the remark this answers is not in the document]
Comment (Namanl2001) [reply .github/pull_request_template.md:36]: Sorry! my bad. Just remove the last one `Any dependent changes have been merged and published in downstream modules.`

[the remark this answers is not in the document]
Comment (Namanl2001) [reply .github/pull_request_template.md]: And don't forget to add `I have verified/tested my code by running it locally.`

[the remark this answers is not in the document]
Comment (kanak22) [reply .github/pull_request_template.md]: I did this

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Namanl2001) [reply .github/pull_request_template.md:22]: Two points are coming to my mind:
- `enhancement in the existing features` or something like that.
- `other`.

What do you mean by `build related changes`?

[the remark this answers is not in the document]
Comment (kanak22) [reply .github/pull_request_template.md:22]: build related changes can be anything that compiling source files. packaging compiled files into compressed formats (such as jar, zip) etc.

[the remark this answers is not in the document]
Comment (Namanl2001) [reply .github/pull_request_template.md:22]: Cool! but I think it would not be useful for this project.

[the remark this answers is not in the document]
Comment (Namanl2001) [reply .github/pull_request_template.md:22]: Okay these two only 🤗

**Review by kanak22 — COMMENTED** [review]
I made the asked changes.

**Review by Namanl2001 — APPROVED** [review]
Thanks @kanak22 . Ready to merge.
Are you a GSSoC'21 participant?

**Comment by kanak22** [comment]
thankyou so much! for giving me an opportunity

**Comment by kanak22** [comment]
Sure I will do it by night 🤗

**Comment by Namanl2001** [comment]
Hey @kanak22 . I really appreciate your enthusiasm to contribute. But it's not urgent, take your time and have some rest. It would be totally fine if you would do it by tomorrow. . Thanks!

**Comment by kanak22** [comment]
Thankyousomuch 😊

**Pull request merged**